srai/loaders/osm_loaders/osm_tile_data_collector.py

```diff
@@ -0,0 +1,103 @@
+"""This module contains classes of strategy for handling downloaded tiles."""
+from abc import ABC, abstractmethod
+from enum import Enum
+from pathlib import Path
+from typing import Any, Union
+
+from PIL import Image
+
+
+class DataCollector(ABC):
+    """Stores collected images."""
+
+    @abstractmethod
+    def store(self, x: int, y: int, data: Image.Image) -> Any:
+        """
+        Apply action for object storage and returns data of it.
+
+        Args:
+            x (int): x coordinate of tile, used as id
+            y (int): y coordinate of tile, used as id
+            data (Image.Image): tile
+        """
```

Comment:
Are `x` and `y` enough to create a unique id? `zoom` is not needed here as well?

Reply:
I assumed single OSMTileLoader object will be used with one zoom so x and y are enough. If we want to be unique with different zooms, maybe store should get id used by SlippyMapRegionizer and this id should have all information?

Reply:
SIngle id is in fact even better, let's do it like that ;)

Reply:
add feature_id as a concat of `x`, `y` and `z`, and add them also as a columns separately